desktop/src/backends/ui.rs

```diff
@@ -136,7 +136,9 @@ impl DesktopUiBackend {
         Ok(Self {
             window,
             cursor_visible: true,
-            clipboard: Clipboard::new().context("Couldn't get platform clipboard")?,
+            clipboard: Clipboard::new()
+                .context("Couldn't get platform clipboard")
```

Comment:
Is this going to be logged anywhere, if followed by an `.ok()` immediately? :thinking:

Reply:
I wrote this code late and I've not done much rust recently so I'm a little out of practice. Sorry.

I guess the "anyhow"s `context` call doesn't do anything in this context.

"arboard" spits out a pretty detailed error if that's sufficient, or I could check if `Clipboard::new().ok().is_none()` and warn.
```
2024-03-12T17:12:26.436744Z  WARN arboard::platform::linux: Tried to initialize the wayland data control protocol clipboard, but failed. Falling back to the X11 clipboard protocol. The error was: Unknown error while interacting with the clipboard: A required Wayland protocol (zwlr_data_control_manager_v1 version 1) is not supported by the compositor
```

Reply:
You could do `Clipboard::new().inspect_err(|err| tracing::error("Failed to initialize clipboard: {err}")).ok()`

Reply:
Neat, thanks for the tip :)